src/pivot-table/components/cells/DimensionTitleCell.tsx

```diff
@@ -89,6 +89,7 @@ const DimensionTitleCell = ({
       background={open ? activeBackground : background}
       hoverBackground={hoverBackground}
       shouldShowMenuIcon={shouldShowMenuIcon}
+      isDimension={isDim}
```

Comment:
Set pointerEvents here instead? No need for the logic in `StyledHeaderCellWrapper`?

```suggestion
      pointerEvents={isDimension ? "all" : "none"}
```

Reply:
Yes, thought of it, but it's not a good practice IMO, since we have all styles in `StyledHeaderCellWrapper`